azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/connecting/overlay/ConnectingOverlayViewModel.kt

```diff
@@ -24,13 +25,17 @@ import kotlinx.coroutines.flow.StateFlow
 internal class ConnectingOverlayViewModel(
     private val dispatch: (Action) -> Unit,
     private val isTelecomManagerEnabled: Boolean = false,
+    private val callType: CallType? = null,
 ) {
 
     private lateinit var displayOverlayFlow: MutableStateFlow<Boolean>
     private lateinit var networkManager: NetworkManager
 
     private lateinit var cameraStateFlow: MutableStateFlow<CameraOperationalStatus>
     private lateinit var audioOperationalStatusStateFlow: MutableStateFlow<AudioOperationalStatus>
+    private var callingStatus: CallingStatus = CallingStatus.NONE
```

Comment:
We should not keep status in the view model. Better to consume from the Redux state.